### PR TITLE
add helper page for git info

### PR DESCRIPTION
**Pull request by beaugogh**

### CODE REVIEW

**Comment by kretep** [comment]
Could it maybe also include the long sha?

File: src/app/containers/help/help.constant.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+angular.module("transmartBaseUiGitConstants", [])
+
+    .constant("gitInfo", {
```
Comment (rnugraha) [inline src/app/containers/help/help.constant.js]: What are these

File: src/app/containers/help/help.constant.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+        "branch": "helper",
+        "tag": "v0.1.1",
+        "message": "refactor helper page",
+        "url": "https://github.com/thehyve/transmart-base-ui/commit/b24762c4eb1e774c7ae2c62f04fcae32dd59bd15",
+        "count": 889
+    })
 
-;
\ No newline at end of file
+;
```
Comment (kretep) [inline src/app/containers/help/help.constant.js]: Isn't this file auto generated? In that case it shouldn't be in git, right?

File: src/app/containers/help/help.constant.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+    .constant("gitInfo", {
+        "short": "b24762c",
+        "long": "b24762c4eb1e774c7ae2c62f04fcae32dd59bd15",
+        "branch": "helper",
+        "tag": "v0.1.1",
+        "message": "refactor helper page",
+        "url": "https://github.com/thehyve/transmart-base-ui/commit/b24762c4eb1e774c7ae2c62f04fcae32dd59bd15",
+        "count": 889
+    })
```
Comment (PiotrZakrzewski) [inline src/app/containers/help/help.constant.js]: If this is generated then how about adding it to .gitignore?

File: src/app/containers/help/help.constant.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+	"short": "72f534d",
+	"long": "72f534d3a4d3193862a3e17eda150bb13cba7989",
+	"branch": "helper",
+	"tag": "v0.1.1",
+	"message": "format help.constant",
+	"url": "https://github.com/thehyve/transmart-base-ui/commit/72f534d3a4d3193862a3e17eda150bb13cba7989",
+	"count": 891
+})
+
+;
```
Comment (rnugraha) [inline src/app/containers/help/help.constant.js]: End of line

File: src/app/containers/help/help.js [hunk 5]
```diff
@@ -14,7 +14,9 @@ angular.module('transmartBaseUi')
                         templateUrl: 'app/containers/help/help.html'
                     },
                     'content@help': {
-                        templateUrl: 'app/containers/help/help.content.html'
+                        templateUrl: 'app/containers/help/help.content.html',
+                        controller: 'HelpCtrl',
+                        controllerAs: 'vm'
```
Comment (kretep) [inline src/app/containers/help/help.js]: Personally, I would have called it 'ctrl' or something :)

Comment (rnugraha) [inline src/app/containers/help/help.js]: not high priority at this point of time

**Comment by beaugogh** [comment]
fixed now, see the changes in server.js and unit-tests.js

**Pull request merged**